pyretis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+#{% set name = "pyretis" %}
+#{% set version = "0.9.2" %}
+#{% set sha256 = "f738210a323dd8f84f98f33f0836dfdcc04f3fa8559a4e847f8eb311f147c7cc" %}
+
+package:
+   name: pyretis
+   version: 0.9.2
+#  name: {{ name|lower }}
+#  version: {{ version }}
+
+source:
```

Comment:
ok, seems that I finally managed to make it work. Thank you tons. 

I would like to add mdtraj in the list of packages, but it fails if I do so. Do you have any tips for that?

Once more, really thanks!